LibreNMS/Alert/Transport/Pagerduty.php

```diff
@@ -54,12 +55,18 @@ public function deliverAlert($obj, $opts)
      */
     public function contactPagerduty($obj, $config)
     {
+        $groups = Device::find($obj['device_id'])->groups;
```

Comment:
You should use `\DeviceCache::get($obj['device_id'])->groups` instead so it doesn't fetch the device from the database again.